pkg/middleware/auth/auth.go

```diff
@@ -65,14 +65,16 @@ func Authenticate(conf *AuthOptions) echo.MiddlewareFunc {
 
 			var au *auth.AuthenticatedUser
 
+			var id string
+
```

Comment:
this is a nit, but if you are declaring multiple variables in a row its usually better to do: 

```suggestion
			var  (
				au *auth.AuthenticatedUser
				id string
			)
```